### PR TITLE
Preventing arithmetic overflow panic

### PR DESCRIPTION
**Pull request by ustulation**
[Review on Reviewable](https://reviewable.io/reviews/maidsafe/routing/397)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @inetic

(maidsafe_highfive has picked a reviewer for you, use r? to override)

**Comment by benjaminbollen** [comment]
can you use this function? 

https://github.com/maidsafe/routing/pull/396/files#diff-f001368a485a2355efa38f1598bd55e4L133

File: src/routing_client.rs [hunk 1]
```diff
@@ -275,9 +275,8 @@ impl<F> RoutingClient<F> where F: Interface {
     }
 
     fn get_next_message_id(&mut self) -> MessageId {
-        let current = self.next_message_id;
-        self.next_message_id += 1;
-        current
+        self.next_message_id = self.next_message_id.wrapping_add(1);
+        self.next_message_id
```
Comment (benjaminbollen) [inline src/routing_client.rs]: please don't undo the current

Comment (benjaminbollen) [inline src/routing_client.rs]: but should not matter; merging

**Comment by benjaminbollen** [comment]
looks good to me

**Pull request merged**